### PR TITLE
Actions

### PR DESCRIPTION
**Pull request by JacobMoller**
Added build and test yml file
Removed .local folder

### CODE REVIEW

File: sql/data/[User].csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+id,name
+,Emma
+,Malchau
+,Jepp
+,Jacob
+,Freja
+,Josefine
```
Comment (frejabv) [inline sql/data/[User].csv:7]: What is this used for?

**Comment by Werenfeldt** [comment]
This PR contains old Docker files. I think we should close this PR and make the Actions in a new one.

File: sql/data/[User].csv (hunk 1 above)
Comment (Werenfeldt) [reply sql/data/[User].csv:7]: Old Docker test. It should not be merged into main :) Might certainly explode.

**Comment by JacobMoller** [comment]
> This PR contains old Docker files. I think we should close this PR and make the Actions in a new one.

Okay, go ahead and copy what is needed and i will close this one afterwards :)

**Comment by Werenfeldt** [comment]
> > This PR contains old Docker files. I think we should close this PR and make the Actions in a new one.
> 
> Okay, go ahead and copy what is needed and i will close this one afterwards :)

See #204 :)

**Pull request closed without merging**